runner.go

```diff
@@ -218,7 +219,7 @@ func (r *Runner) Stop() {
 func (r *Runner) Receive(d dep.Dependency, data interface{}) {
 	r.dependenciesLock.Lock()
 	defer r.dependenciesLock.Unlock()
-	log.Printf("[DEBUG] (runner) receiving dependency %s", d)
+	namedLogger("runner").Debug("receiving dependency %s", d)
```

Comment:
Want to double check if `%s` will print out the string for `d`? Does the string need to be wrapped in a `Sprintf` or alternatively not use `%s` or does the logger work in a different way?

Reply:
It would work as the Dependency type (ie. d) has String() as part of the interface. The implementation returns the ID.

Do you think it would read better changing the `d` to `d.String()`, making that explicit?

Reply:
:neutral_face::palm_tree: 

Duh. Though I think I'll just use a '+ d.String()' vs the other call. Simpler here IMO as it is just tacked onto the end.

Reply:
I meant 'Duh' to myself in case that wasn't clear. :)

Reply:
And I see that I missed that in a couple other places. Going to double check and fix any others as well. Thanks.

Reply:
Oh... and it doesn't need the `+`. Those work like Println() and append each thing anyways.

Reply:
I think I got them all. Thanks again. :heart:

Reply:
I get you! Thanks for chatting about this! ❤️